worldedit-bukkit/src/main/java/com/fastasyncworldedit/bukkit/regions/GriefPreventionFeature.java

```diff
@@ -26,10 +27,11 @@ public GriefPreventionFeature(final Plugin griefpreventionPlugin) {
     }
 
     public boolean isAllowed(Player player, Claim claim, MaskType type) {
-        return claim != null && (claim.getOwnerName().equalsIgnoreCase(player.getName()) || claim
+        return TaskManager.taskManager().sync(() -> claim != null && (claim.getOwnerName().equalsIgnoreCase(player.getName()) || claim
                 .getOwnerName()
                 .equals(player.getUniqueId()) ||
-                type == MaskType.MEMBER && claim.allowBuild(player, Material.AIR) == null);
+                type == MaskType.MEMBER && claim.allowBuild(player, Material.AIR) == null));
```

Comment:
Is it better worth putting the sync only on the final "or" of this as the rest doesn't need the added overhead of executing on main